### PR TITLE
Livepeer.Cloud SPE Milestone #1 Deliverables: Add Gateway Docs

### PR DESCRIPTION
**Pull request by mikezupper**

### CODE REVIEW

File: gateways/guides/linux-install.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+sudo tar -zxvf livepeer-linux-amd64.tar.gz
+sudo rm livepeer-linux-amd64.tar.gz
+sudo mv livepeer-linux-amd64/* /usr/local/bin/
+```
+# Generate a new keystore file
+```
+/usr/local/bin/livepeer -network arbitrum-one-mainnet -ethUrl <ARBITRUM RPC URL> -gateway
+exit
+```
+**When generating a new keystore file, the program will prompt you for a password. This password is used to decrypt the keystore file and access the private key. Make sure to never share or lose access to either the password or the keystore file**
```
Comment (rickstaa) [inline gateways/guides/linux-install.mdx:21]: Could maybe be replaced by callout box to make it standout more.

File: gateways/guides/windows-install.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+PAUSE
+```
+## Start the Livepeer Gateway
+
+Start the Livepeer Gateway using the .bat file.
+
+When prompted enter and confirm a password. 
+
+**This password is used to decrypt the keystore file and access the private key. Make sure to never share or lose access to either the password or the keystore file**
```
Comment (rickstaa) [inline gateways/guides/windows-install.mdx:27]: Maybe callout box?

File: gateways/guides/fund-gateway.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  bridging the funds to Arbritrum's L2 Network, and finally using the Livepeer
+  CLI to allocate the proper deposit and reserve amounts.
+---
+
+# Add Funds to Gateway Wallet
+
+In order to use the Gateway you need to send ETH to your Gateway address
+on Ethereum Mainnet and then bridged to Arbitrum's L2 Network.
+
+**_Note:_**
```
Comment (rickstaa) [inline gateways/guides/fund-gateway.mdx:16]: Callout box?

**Review by rickstaa — CHANGES_REQUESTED** [review]

File: gateways/guides/gateway-overview.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+[Fund the Livepeer Gateway](/gateways/guides/fund-gateway)
+
+[Publish Content](/gateways/guides/publish-content)
+
+[Playback Content](/gateways/guides/playback-content)
+
+## Prerequisites
+
+This guide assumes you are familiar with installing software on unix-based
```
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:31]: This looks fine to me, but what did you have in mind?

File: gateways/guides/gateway-overview.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+systems. You should be comfortable with command-line syntax, Networking,
+Firewalls, and containerized environments. You should have a basic understanding
+of the Livepeer protocol. You dont have to be an expert, but troubleshooting
+skills will come in handy.  For more information view the go Livepeer [installation guide.](//orchestrators/guides/install-go-livepeer)
+
+
```
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:37]: Is this plugin already part of the repo? if so, should this run against all the mdx files?

File: gateways/guides/docker-install.mdx [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+you are running Docker 20.10.x. If an older version of docker is installed
+remove it with the following command:
+
+As the root user (or sudo), run the following:
+
+```
+apt remove docker*
+```
+
+Install Docker
```
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/docker-install.mdx:38]: I would agree that steps make it look better, but we followed the Orchestrator Installation Guide formatting. Is this something you feel is critical to change?

File: gateways/guides/fund-gateway.mdx (hunk 3 above)
Comment (mikezupper) [reply gateways/guides/fund-gateway.mdx:16]: DONE

File: gateways/guides/windows-install.mdx (hunk 2 above)
Comment (mikezupper) [reply gateways/guides/windows-install.mdx:27]: DONE

File: gateways/guides/linux-install.mdx (hunk 1 above)
Comment (mikezupper) [reply gateways/guides/linux-install.mdx:21]: DONE

File: gateways/guides/gateway-overview.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:37]: BTW, I fixed the spacing in this file manually.

File: gateways/guides/gateway-overview.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+description:
+    This guide will walk you through the Livepeer Gateway installation and
+  setup. Steps to install for Ubuntu Linux, Docker, and Windows are provided.
+  Choose the environment you want, follow install instructions, then continue to
+  the configuration section. 
+  
+ 
+icon: Rocket
+---
+ *Note the Livepeer Gateway was previously called the Livepeer Broadcaster so you will see some commands and labels still use the Broadcaster name that haven't been updated in the code.*
```
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:12]: DONE

File: gateways/guides/gateway-overview.mdx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+# Quick Links
+
+[Docker Installation](/gateways/guides/docker-install)
```
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:15]: DONE

File: gateways/guides/docker-install.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (rickstaa) [reply gateways/guides/docker-install.mdx:38]: Ah I see 👍🏻. Let's leave that decision to @adamsoffer.

File: gateways/guides/gateway-overview.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (rickstaa) [reply gateways/guides/gateway-overview.mdx]: It is not part of the repo. I have it in Vscode. I am not sure if the team already uses prettier for the docs. Let's see what @adamsoffer has to say about this 👍🏻.

File: gateways/guides/gateway-overview.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (mikezupper) [reply gateways/guides/gateway-overview.mdx:40]: We modified the text and updated the pull request.

File: gateways/guides/gateway-overview.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (adamsoffer) [reply gateways/guides/gateway-overview.mdx]: Yeah I'm cool with running prettier's mdx plugin against all the mdx files 👍

[the remark this answers is not in the document]
Comment (rickstaa) [reply gateways/guides/gateway-overview.mdx]: @mikezupper I created a pull request with the prettier suggestions. Feel free to merge if you think it improves the readability -> https://github.com/mikezupper/livepeer-docs/pull/3/files.

**Comment by rickstaa** [comment]
@mikezupper, I just realized that there isn’t a general node explanation on the [livepeer-network](https://github.com/mikezupper/livepeer-docs/blob/d1fd9379e47b288f1c0f76fcb64237343fa679e5/developers/core-concepts/livepeer-network/delegators.mdx#L13) page. Could we possibly add that as well?

**Review by thomshutt — APPROVED** [review]

**Review by rickstaa — APPROVED** [review]

**Pull request merged**